images/hook-ip/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM linuxkit/alpine:146f540f25cd92ec8ff0c5b0c98342a9a95e479e AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add curl
+RUN apk add --no-cache --initdb -p /out \
+    alpine-baselayout \
+    bash \
+    busybox \
+    iproute2 \
+    iptables \
+    ebtables \
+    ipvsadm \
+    bridge-utils \
+    musl \
+    sed
+
+# Remove apk residuals
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM scratch
+ENTRYPOINT []
+CMD []
+WORKDIR /
+COPY --from=mirror /out/ /
```

Comment:
Sorry, why do we need this new image? i dont see any of these tools being used?

Reply:
Indeed. I just copied `linuxkit/ip` -- previously used directly -- from https://github.com/linuxkit/linuxkit/blob/master/pkg/ip/Dockerfile -- just to add `sed` (GNU sed) required for proper kernel cmdline parsing in the bash code. I discovered busybox's sed didn't really work when I tried actually passing a `vlan_id=876` parameter and that failed.

As for the tools: I kept the original ones we had there, minus wireguard. I agree that probably most of them aren't used either.